tests/nested/manual/fakeinstaller-real/task.yaml

```diff
@@ -0,0 +1,253 @@
+      exit
+  fi
+  #shellcheck source=tests/lib/store.sh
+  . "$TESTSLIB"/store.sh
+  teardown_fake_store "$STORE_DIR"
+  rm -rf ./classic-root
+
+execute: |
+  if [ "$TRUST_TEST_KEYS" = "false" ]; then
+      echo "This test needs test keys to be trusted"
+      exit
+  fi
+  # shellcheck source=tests/lib/prepare.sh
+  . "$TESTSLIB/prepare.sh"
+  #shellcheck source=tests/lib/nested.sh
+  . "$TESTSLIB"/nested.sh
+
+  # install the snapd deb from spread so we are using the same version to 
+  # validate the seed as well as call preseed, etc. 
+  dpkg -i "$SPREAD_PATH"/../snapd_*.deb
+
+  echo Expose the needed assertions through the fakestore
+  cp "$TESTSLIB"/assertions/developer1.account "$STORE_DIR/asserts"
+  cp "$TESTSLIB"/assertions/developer1.account-key "$STORE_DIR/asserts"
+  cp "$TESTSLIB"/assertions/testrootorg-store.account-key "$STORE_DIR/asserts" 
+  export SNAPPY_FORCE_SAS_URL=http://$STORE_ADDR
+
+  version="$(nested_get_version)"
+
+  # build updated shim
+  version=22
+  snap download --basename=pc --channel="$version/edge" pc
+  unsquashfs -d pc-gadget pc.snap
+  echo 'console=ttyS0 systemd.journald.forward_to_console=1' > pc-gadget/cmdline.extra
+  echo "Sign the shim binary"
+  KEY_NAME=$(tests.nested download snakeoil-key)
+  SNAKEOIL_KEY="$PWD/$KEY_NAME.key"
+  SNAKEOIL_CERT="$PWD/$KEY_NAME.pem"
+  tests.nested secboot-sign gadget pc-gadget "$SNAKEOIL_KEY" "$SNAKEOIL_CERT"
+  snap pack --filename=pc.snap pc-gadget/ 
+
+  # get an updated kernel
+  snap download --basename=pc-kernel --channel="$version/edge" pc-kernel
+  uc20_build_initramfs_kernel_snap "$PWD/pc-kernel.snap" "$NESTED_ASSETS_DIR"
+  mv "${NESTED_ASSETS_DIR}"/pc-kernel_*.snap pc-kernel.snap
+
+  # prepare a classic seed
+  # TODO:
+  # - create pc-classic custom gadget
+  # - repacked snapd snap
+  # (should be as simple as addinga "--snap=./local-gadget.snap ...")
+  LABEL="$(date +%Y%m%d)"
+  gendeveloper1 sign-model < "$TESTSLIB"/assertions/developer1-22-classic-dangerous.json > my.model
+  snap prepare-image --classic \
+      --channel=edge \
+      --snap ./pc-kernel.snap \
+      --snap ./pc.snap \
+      my.model \
+      ./classic-seed
+  # make the seed label more predictable for fake-installer auto-mode
+  NEW_LABEL=classic
+  mv ./classic-seed/system-seed/systems/"$LABEL" ./classic-seed/system-seed/systems/"$NEW_LABEL"
+  LABEL="$NEW_LABEL"
+
+  # we don't need the fakestore anymore
+  teardown_fake_store "$STORE_DIR"
+
+
+  # build the fake-installer snap
+  snap install snapcraft --candidate --classic
+  snap install lxd --candidate
+  snap set lxd waitready.timeout=240
+  lxd waitready
+  lxd init --auto
+  echo "Setting up proxy for lxc"
+  if [ -n "${http_proxy:-}" ]; then
+    lxd.lxc config set core.proxy_http "$http_proxy"
+  fi
+  if [ -n "${https_proxy:-}" ]; then
+    lxd.lxc config set core.proxy_https "$http_proxy"
+  fi
+  (cd "$TESTSLIB"/fakeinstaller && snapcraft)
+  FAKEINSTALLER_SNAP="$(ls "$TESTSLIB"/fakeinstaller/*.snap)"
```

Comment:
Nitpick (and to avoid shellcheck complaining: `find "$TESTSLIB"/fakeinstaller/ -maxdepth 1 -name '*.snap'`